src/common/tracing.ts

```diff
@@ -1,18 +1,147 @@
 import { Tracing } from '@map-colonies/telemetry';
-import { IGNORED_INCOMING_TRACE_ROUTES, IGNORED_OUTGOING_TRACE_ROUTES } from './constants';
+import config from 'config';
+import { type Attributes, type Span, type SpanOptions, SpanStatusCode, trace } from '@opentelemetry/api';
+import { IGNORED_INCOMING_TRACE_ROUTES, IGNORED_OUTGOING_TRACE_ROUTES, SERVICE_NAME, schemasPackageVersion } from './constants';
 
-export const tracing = new Tracing({
+function maskSchemasPath(path: string): string {
+  const split = path.split('node_modules/');
+  if (split.length > 1) {
+    return split[1];
+  }
+  return '';
+}
+
+const fsFunctionMap = new Map([
```

Comment:
Consider adding more useful functions

Reply:
After checking again, you specified the right functions except for "readFile", it should be "readFileSync"

Reply:
readFileSync is only used in startup
readFile is used by the loadSchema